phi.float/src/datapack/functions/add.mcfunction

```diff
@@ -0,0 +1,2 @@
+summon minecraft:armor_stand -30000000 0 1600 {Marker:1b,Invisible:1b,Tags:["J0SfmBoy"],Attributes:[{Name:"generic.maxHealth",Base:199999999999999999999999999999999999999d}],ArmorItems:[{id:"minecraft:stone",Count:1b,tag:{AttributeModifiers:[{AttributeName:"generic.maxHealth", Amount:0.0, Operation:0, UUIDLeast:1, UUIDMost:1}]}},{id:"minecraft:stone",Count:1b,tag:{AttributeModifiers:[{AttributeName:"generic.maxHealth", Amount:0.0, Operation:0, UUIDLeast:2, UUIDMost:1}]}}]}
+execute as @e[limit=1,tag=J0SfmBoy] function phi.float/internal/add_resume
```

Comment:
Should be phi.float:internal/add_resume, please make sure you test all functions before submitting them